index.src.html

```diff
@@ -409,6 +409,12 @@ <h3 id="ip-address-space-heading">IP Address Space</h3>
         <td>[[RFC6598]]</td>
         <td>[=IP address space/local=]</td>
       </tr>
+      <tr>
+        <td>`198.18.0.0/15`</td>
+        <td>Benchmarking</td>
+        <td>[[RFC2544]]</td>
+        <td>[=IP address space/local=]</td>
+      </tr>
```

Comment:
Mind moving this one row lower? I tried to keep the table sorted lexicographically by (IP address space, IP address range).

Reply:
Done, although I don't quite understand the sorting order (e.g. why 198.18 should be above 192.168).

Also reached out to get the IPR issue resolved. Will post an update when that's done.

Reply:
Aha, you're right. I read 192.18... And the link-local address range right below is not sorted correctly. Oh well. Good enough as is.

Reply:
Ah, that makes more sense. Moved it below the 192.168 range.